meta/store.go

```diff
@@ -997,7 +996,7 @@ func (s *Store) Databases() (dis []DatabaseInfo, err error) {
 }
 
 // CreateDatabase creates a new database in the store.
-func (s *Store) CreateDatabase(name string) (*DatabaseInfo, error) {
+func (s *Store) CreateDatabase(name string, retentionPeriod time.Duration) (*DatabaseInfo, error) {
```

Comment:
Just to we're clear, @pauldix is saying this should be RetentionPolicyInfo type, not just a duration.